hygen/templates/entity/new/Entity.ejs.t

```diff
@@ -5,28 +5,29 @@ import AccountTreeIcon from '@mui/icons-material/AccountTree';
 import EntityInterface from '@irontec/ivoz-ui/entities/EntityInterface';
 import _ from '@irontec/ivoz-ui/services/translations/translate';
 import defaultEntityBehavior from '@irontec/ivoz-ui/entities/DefaultEntityBehavior';
+import { EntityValue } from '@irontec/ivoz-ui';
 import selectOptions from './SelectOptions';
 import Form from './Form';
 import { foreignKeyGetter } from './ForeignKeyGetter';
-import { <%= Name %>Properties } from './<%= Name %>Properties';
+import { <%= Name %>Properties, <%= Name %>PropertyList } from './<%= Name %>Properties';
 import foreignKeyResolver from './ForeignKeyResolver';
 
 const properties: <%= Name %>Properties = {
 <%- h.formattedEntity(h.url(), Name)%>
 };
 
-const <%= name %>: EntityInterface = {
+const <%= Name %>: EntityInterface = {
   ...defaultEntityBehavior,
   icon: AccountTreeIcon,
   iden: '<%= Name %>',
   title: _('<%= Name %>', { count: 2 }),
   path: '/<%= h.inflection.pluralize(name) %>',
-  toStr: (row: any) => row.id,
+  toStr: (row: <%= Name %>PropertyList<EntityValue>) => row.name as string,
```

Comment:
You can't know if this entity has name property. Use `row.id.toString()` or something like that